packages/shopping/src/__tests__/acceptance/helper.ts

```diff
@@ -27,3 +27,7 @@ export async function setupApplication(): Promise<AppWithClient> {
 
   return {app, client};
 }
+
+export function getExpiredToken(): string {
+  return 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjEiLCJuYW1lIjoiRXhhbXBsZSBVc2VyIiwiaWF0IjoxNTU3NTE1MTg1LCJleHAiOjE1NTc1MTUyNDV9.NZ31amMztvIYBth3SRY9fxiZTukObpgV2gSmJZ10pwY';
```

Comment:
Is there a way to use `jsonwebtoken ` api to create an expired token? If not, at least document how it was created at the first place with the corresponding json value.

Reply:
@raymondfeng , using the utility, I created a token that expires in 5 minutes and printed out its value in console.log. Then, every other time I use it, it is always expired.

packages/shopping/src/services/jwt-service.ts is used to create the jwt token:
```
  async generateToken(userProfile: UserProfile): Promise<string> {
    if (!userProfile) {
      throw new HttpErrors.Unauthorized(
        'Error generating token : userProfile is null',
      );
    }

    // Generate a JSON Web Token
    let token: string;
    try {
      token = await signAsync(userProfile, this.jwtSecret, {
        expiresIn: Number(this.jwtExpiresIn),
      });
    } catch (error) {
      throw new HttpErrors.Unauthorized(`Error encoding token : ${error}`);
    }

    return token;
  }
```

Reply:
packages/shopping/src/__tests__/acceptance/user.controller.acceptance.ts now contains :
givenAnExpiredToken()